Menu/Menu.js

```diff
@@ -1,10 +1,12 @@
 
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open');
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector(".menu");
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelectorAll(".menu-button");
```

Comment:
Remember, querySelectorAll returns a node list instead of a node. So you could change this to:

```suggestion
const menuButton = document.querySelector(".menu-button");
```